pages/common/pydocstyle.md

```diff
@@ -0,0 +1,32 @@
+# pydocstyle
+
+> A static analysis tool for checking compliance with Python docstring conventions.
+> More information: <https://www.pydocstyle.org/en/latest/>.
+
+- Analyze a Python script file or all the Python scripts inside a specific directory:
+
+`pydocstyle {{file.py|path/to/directory}}`
+
+- Show [e]xplanation of each error:
+
+`pydocstyle -e | --explain {{file.py|path/to/directory}}`
+
+- Show [d]ebug information:
+
+`pydocstyle -d | --debug {{file.py|path/to/directory}}`
+
+- Count and print the total number of errors to stdout:
+
+`pydocstyle --count {{file.py|path/to/directory}}`
+
+- Use the given config file and disable config discovery:
+
+`pydocstyle --config={{path/to/config_file}} {{file.py|path/to/directory}}`
+
+- Specified which errors to ignore out of all of the available error codes (with a list of comma-separated error codes or prefixes). for example: `--ignore=D101,D2`:
```

Comment:
```suggestion
- Ignore one or more errors:
```
We can be even more direct, some time ago I did some PRs that removed "specify" on some pages when it was possible to avoid it and also left "a comma-separated list" implicit, being demonstrated in the list placeholder.

Reply:
I agree

Reply:
Hello @tricantivu and @vitorhcl.

Same as above, I will opt for the more simplistic description.

Reply:
Done.